pkgs/development/python-modules/attacut/default.nix

```
@@ -0,0 +1,73 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  fetchpatch,
+
+  setuptools,
+
+  pytestCheckHook,
+
+  docopt,
+  fire,
+  numpy,
+  python-crfsuite,
+  pyyaml,
+  six,
+  ssg,
+  torch,
+}:
+
+buildPythonPackage rec {
+  pname = "attacut";
+  version = "1.1.0-dev";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "PyThaiNLP";
+    repo = "attacut";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-x3JJC1Xd+tsOAHJEHGzIrhIrNGSvLSanAFc7+uXb2Kk=";
+  };
+
+  patches = [
+    # this patch is required to apply the following patch
+    (fetchpatch {
+      name = "update-setup-py.patch";
+      url = "https://github.com/PyThaiNLP/attacut/commit/c3d430ae42bbf22b000da449c6f045b78442f88e.patch";
+      hash = "sha256-iVkW50bdSVKZLq/lOcgrfk7gVDmZzqJb6gybMnm2Gw0=";
+    })
+    (fetchpatch {
+      name = "fix-nptyping-deprecated-array.patch";
+      url = "https://github.com/PyThaiNLP/attacut/commit/a707297b3f08a015d32d8ac241aa8cb11128cbd4.patch";
+      hash = "sha256-pIVxwitYM+7+6c2236R7GTnkgXKPVtK/SkdhGw+1Gb4=";
+    })
```

```suggestion
    (fetchpatch {
      name = "fix-nptyping-deprecated-array.patch";
      url = "https://github.com/PyThaiNLP/attacut/commit/a707297b3f08a015d32d8ac241aa8cb11128cbd4.patch";
      includes = [ "attacut/evaluation.py" ];
      hash = "sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=";
    })
```
I guess it works if we only get the patch we really need.

---

@natsukium ok, i remove no more need nptyping module with sed...